cmd/sidecar/local-server.go

```diff
@@ -182,6 +182,9 @@ func (srv *LocalServer) logHandler(w http.ResponseWriter, r *http.Request) {
 	_, err := srv.flow.ActionLog(req.ctx, &grpc.ActionLogRequest{
 		InstanceId: req.instanceId,
 		Msg:        []string{msg},
+		ActionID:   req.actionId,
+		Originator: req.originator,
```

Comment:
`originator` is redundant now i can/will remove it, it was meant initially to solve the log inheritance problem.  
`iterator` is an identifier for child-instances that where created in a for each loop so that for expl. we can select the logs of the second iteration of a forloop